server/picture-profile.json

```diff
@@ -0,0 +1,196 @@
+{
+    "editor": {
+        "language": {
+            "sdWidth": "half",
+            "order": 3,
+            "section": "header",
+            "required": false
+        },
+        "priority": null,
+        "news_item_types": null,
+        "headline": {
+            "order": 5,
+            "required": false,
+            "formatOptions": [],
+            "section": "header"
+        },
+        "usageterms": null,
+        "anpa_take_key": null,
+        "iptc_subject_codes": null,
+        "news_products": null,
+        "services-products": {
+            "order": 1,
+            "section": "header",
+            "required": false
+        },
+        "byline": null,
+        "sms": null,
+        "media_description": null,
+        "ednote": null,
+        "genre": null,
+        "sign_off": null,
+        "abstract": null,
+        "company_codes": null,
+        "dateline": null,
+        "body_footer": null,
+        "languages": null,
+        "belga_related_articles": null,
+        "distribution": {
+            "order": 2,
+            "section": "header",
+            "required": false
+        },
+        "belga-url": null,
+        "credits": null,
+        "place": null,
+        "attachments": null,
+        "author_roles": null,
+        "belga_related_images": null,
+        "news_services": null,
+        "slugline": {
+            "sdWidth": "full",
+            "order": 4,
+            "section": "header",
+            "required": false
+        },
+        "feature_media": null,
+        "urgency": null,
+        "body_html": null,
+        "footer": null,
+        "label": null,
+        "anpa_category": null,
+        "people": {
+            "order": 7,
+            "section": "header"
+        },
+        "event_description": {
+            "order": 8,
+            "section": "header",
+            "sdWidth": "full",
+            "textarea": true
+        },
+        "source": {
```

Comment:
if `source` is always "Belga" and filled with extension anyway, I would remove it from the editor.

Reply:
I think if it's not there it won't make it to the formatter

Reply:
I've tried and it worked. I hope it was not cache :)

Reply:
ok thx for checking